src/tests/Dashboard.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import { shallow, mount } from 'enzyme';
+import DashBoard from '../components/DashBoard';
+import presetThemes from '../data/preset-themes';
+
+
+describe('DashBoard', () => {
+  let wrapper;
+
+  const aubergine = presetThemes[0].themeColor;
+  const lastTheme = presetThemes[8].themeColor;
```

Comment:
The constant `lastTheme` is never used.

Reply:
To make this test a more independent, can you use some test data instead of pulling it from preset-themes?

`const aubergineTheme = ["#3F0E40", "#350d36", "#1164A3", "#FFFFFF", "#350D36", "#FFFFFF", "#2BAC76", "#CD2553"];`